### PR TITLE
Work off to-do's from previous PR

### PR DESCRIPTION
**Pull request by johnkerl**
As in title

### CODE REVIEW

**Comment by johnkerl** [comment]
The first three commits are cosmetic & a compile suffices.

Test plan for the fourth:

**Post a new threat descriptor using TMK hash type:**

```
hash="../tmk/sample-hashes/chair-22-sd-sepia-bar.tmk"
description="testing te-tag-query with post"
whitelist_app="1064060413755420"

java com.facebook.threatexchange.TETagQuery \
  -s \
  -a TX_ACCESS_TOKEN_MHST \
  submit \
    -i $hash \
    -t HASH_TMK \
    -d "$description" \
    -l AMBER \
    -s MALICIOUS \
    -p HAS_WHITELIST \
    --privacy-members 1064060413755420 \
    --tags testing_java_post
```
That prints the ID of the newly created descriptor: `2290659920988866`

**Retrieve the new descriptor:**

```
$ echo 2290659920988866 | java com/facebook/threatexchange/TETagQuery -a TX_ACCESS_TOKEN_MHST -s -v ids-to-details --hash-dir checkit

$ ls checkit/
2290659920988866.tmk
```

**Check against original file:**

```
$ ../tmk/cpp/tmk-two-level-score checkit/*.tmk ../tmk/sample-hashes/*sepia*tmk
1.000000 1.000000 ../tmk/sample-hashes/chair-22-sd-sepia-bar.tmk checkit/2290659920988866.tmk

$ md5sum ../tmk/sample-hashes/*sepia* ./checkit/2290659920988866.tmk
86dbb18c680ef222fd42032865ee68f3  ../tmk/sample-hashes/chair-22-sd-sepia-bar.tmk
86dbb18c680ef222fd42032865ee68f3  ./checkit/2290659920988866.tmk
```
This shows that an on-disk file was encoded and uploaded, then downloaded and decoded, recovering the original file contents.

File: hashing/te-tag-query-java/com/facebook/threatexchange/TETagQuery.java [hunk 1]
```diff
@@ -976,6 +899,11 @@ public void handle(
         } else if (option.equals("--dry-run")) {
           dryRun = true;
 
+        } else if (option.equals("-I")) {
```
Comment (mounikabodapati) [inline hashing/te-tag-query-java/com/facebook/threatexchange/TETagQuery.java]: Maybe I'm missing something, I don't see a definition for the -I in the usage() function.

Comment (johnkerl) [reply hashing/te-tag-query-java/com/facebook/threatexchange/TETagQuery.java]: thx

**Comment by johnkerl** [comment]
Test plan for the 5th:

**MD5 multi-post:**

```
description="testing te-tag-query with post"
whitelist_app="1064060413755420"

java com.facebook.threatexchange.TETagQuery \
  -s -a TX_ACCESS_TOKEN_MHST \
  submit \
    -I \
    -t HASH_MD5 \
    -d "$description" \
    -l AMBER \
    -s MALICIOUS \
    -p HAS_WHITELIST \
    --privacy-members 1064060413755420 \
    --tags testing_java_post \
<<EOF
14758f1afd44c09b7992073ccf00b001
14758f1afd44c09b7992073ccf00b002
14758f1afd44c09b7992073ccf00b003
14758f1afd44c09b7992073ccf00b004
EOF
```

followed by

```
$ java com/facebook/threatexchange/TETagQuery -a TX_ACCESS_TOKEN_MHST tag-to-details --hash-type md5 testing_java_post
{"tag_name":"testing_java_post","tag_id":"2911371008904239"}
{"page_index":"0","num_items":"24"}
...
{"id":"2275657035816272","type":"THREAT_DESCRIPTOR","hash":"14758f1afd44c09b7992073ccf00b001"}
{"id":"2596794767007774","type":"THREAT_DESCRIPTOR","hash":"14758f1afd44c09b7992073ccf00b002"}
{"id":"2731786353562359","type":"THREAT_DESCRIPTOR","hash":"14758f1afd44c09b7992073ccf00b003"}
{"id":"2263564343760998","type":"THREAT_DESCRIPTOR","hash":"14758f1afd44c09b7992073ccf00b004"}
```

**TMK multi-post:**

```
description="testing te-tag-query with post"
whitelist_app="1064060413755420"

if [ $# -eq 1 ]; then
  hash="$1"
fi

ls ../tmk/sample-hashes/*.tmk \
| java com.facebook.threatexchange.TETagQuery \
  -s \
  -a TX_ACCESS_TOKEN_MHST \
  submit \
    -I \
    -t HASH_TMK \
    -d "$description" \
    -l AMBER \
    -s MALICIOUS \
    -p HAS_WHITELIST \
    --privacy-members 1064060413755420 \
    --tags testing_java_post
```

followed by

```
$ java com/facebook/threatexchange/TETagQuery -a TX_ACCESS_TOKEN_MHST tag-to-details --hash-type tmk --hash-dir checkmulti testing_java_post
...

$ ls -l checkmulti
total 8320
-rw-r--r--  1 263344 Aug  8 22:34 1207937922663382.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2186246798168803.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2220792231362951.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2269129013216881.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2290659920988866.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2500816413316637.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2545907358795219.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2598281126895395.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2615909071777193.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2688401551179793.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2843065875722581.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2916499501756126.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2932044730145476.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2936405979721904.tmk
-rw-r--r--  1 263344 Aug  8 22:34 2951599224910666.tmk
-rw-r--r--  1 263343 Aug  8 22:34 3536292243063224.tmk
```
```

**Comment by johnkerl** [comment]
Sixth commit adds `-I` to `submit` `usage`

**Pull request merged**